app/controllers/application_controller.rb

```diff
@@ -26,6 +28,10 @@ def authenticated?
     session["authentication"].present?
   end
 
+  def govuk_account_layout?
+    defined?(use_govuk_account_layout?) && use_govuk_account_layout? == true
+  end
```

Comment:
Is there an advantage of defining this `govuk_account_layout?` over having:

```ruby
def use_govuk_account_layout?
  false
end
```

here?